doc/override-pckg.md

```diff
@@ -0,0 +1,44 @@
+Package override
+================
+
+Buildroot typically functions by downloading a tarball, then going through the steps of extracting, configuring, compiling, and installing the software component. The source code is unpacked into a temporary directory named output/build/\<package\>-\<version\>. This directory is ephemeral; it gets completely deleted when the command 'make clean' is executed and is recreated during the next 'make' command execution. Regardless of whether the package source code comes from a Git or Subversion repository, Buildroot first converts it into a tarball and then processes it as it would with any other tarball.
```

Comment:
Suggest using `output/build/<package>-<version>` instead, this way you don't have to escape the ><, and it gives emphasis to the path like we do elsewhere in the documentation.

This is a recurring thing below, including cases where `_` is used, which has syntactic meaning in itself in Markdown, so I suggest using back ticks for all these cases.  

Also, the (CONFIG_DIR)/local.mk is better written `$(CONFIG_DIR)/local.mk`, since `CONFIG_DIR` is a variable.